app/src/main/java/com/ustwo/boilerplate/base/BasePresenter.java

```diff
@@ -41,7 +40,6 @@ public void onViewWillShow(@NonNull final T view) {
      */
     @CallSuper
     public void onViewWillHide() {
```

Comment:
I say either remove everything or keep the disposables and the methods. Don't like half measures. That being said, your logic makes sense @mshipton.